### PR TITLE
Feature/text emphasis on header

### PR DESCRIPTION
**Pull request by MylesB93**
Ability to add text emphasis to Heading field as done here: https://dev.azure.com/etch/DeepSilver.Website/_git/DeepSilver.Website/commit/29b9cd3f5b0e052105870d1fe6fde7737328d1ba?refName=refs%2Fheads%2Fmaster

### CODE REVIEW

**Comment by Newnab** [comment]
Other than my whitespace request I think @peterkeating should take a look at the specifics and make sure he's happy. :)

File: Migrations.cs [hunk 1]
```diff
@@ -512,5 +513,22 @@ public int UpdateFrom11()
...
+        {
+            _contentDefinitionManager.AlterPartDefinition(nameof(Heading), builder => builder
+                .WithField(nameof(Heading.Emphasize), field => field
+                    .OfType(nameof(Fields.Values.Fields.ValuesField))
+                    .WithSettings(new ValuesFieldSettings
+                    {
+                        EmptyMessage = "Currently no text will be emphasized.",
+                        Hint = "Content within the heading text that should be given emphasis.",
+                        NewItemPlaceholder = "Text matching is case sensitive."
+
```
Comment (peterkeating) [inline Migrations.cs:527]: Remove this empty line.

File: Migrations.cs [hunk 2]
```diff
@@ -512,5 +513,22 @@ public int UpdateFrom11()
...
+                    })
+                    .WithDisplayName("Emphasize")
+                    .WithPosition("5")));
+            return 13;
```
Comment (peterkeating) [inline Migrations.cs:531]: Knock this down a line to have an empty line between the migration code and the return.

**Review by peterkeating — CHANGES_REQUESTED** [review]

**Comment by peterkeating** [comment]
> Other than my whitespace request I think @peterkeating should take a look at the specifics and make sure he's happy. :)

@MylesB93 Couple of formatting changes to make the code consistent 👍

**Pull request merged**